release.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -e
+
+declare newversion
+declare current_version
+declare sure
+
+current_version="$(grep -E version ./appium/version.py | sed 's,version *= *.\(\([0-9]*[.]*\)\{3\,4\}\).,\1,g')"
+echo -en "The current version is \033[1;33m$current_version\033[0m, type a new one\n"
+echo -en "\033[1;32mnew version:\033[0m \n"
+read -r newversion
+
+######
+function update_files () {
+    sed -i.back -e "s,$current_version,$newversion,g" appium/version.py
+}
+
+function commit_version_code () {
+    if [ $DRY_RUN ]; then
+        echo -en "\033[1;31m[DRY_RUN]\033[0m call 'git commit ./appium/version.py -m \"Bump $newversion\"'\n"
+    else
+        git commit ./appium/version.py -m "Bump $newversion"
+    fi
+}
+
+function tag_and_generate_changelog () {
+    if [ $DRY_RUN ]; then
+        echo -en "\033[1;31m[DRY_RUN]\033[0m call 'git tag \"v${newversion}\"', generate CHANGELOG.rst and" \
+                 "'git commit CHANGELOG.rst -m \"Update changelog for $newversion\"'\n"
+    else
+        git tag "v${newversion}"
+        gitchangelog > CHANGELOG.rst
+        git commit CHANGELOG.rst -m "Update changelog for $newversion"
+    fi
+}
+
+function upload_sdist () {
+    if [ $DRY_RUN ]; then
+        echo -en "\033[1;31m[DRY_RUN]\033[0m call 'twine upload \"dist/Appium-Python-Client-$newversion.tar.gz\"'\n"
+    else
+        twine upload "dist/Appium-Python-Client-$newversion.tar.gz"
+    fi
+}
+
+function push_changes_to_master () {
+    echo "Push changes and the tag to the master"
+    if [ $DRY_RUN ]; then
+        echo -en "\033[1;31m[DRY_RUN]\033[0m call 'git push origin master' and git 'push origin \"v$newversion\"'\n"
+    else
+        git push origin master
+        git push origin "v$newversion"
```

Comment:
Do you mean `git push --tags`?
I'd have liked to push only a new tag here since `--tags` pushes all of the tags in our local, so I selected pushing one tag way.